### DISCUSSION TITLE
【AI藝術會不會取代人類藝術家？】

### DISCUSSION THREAD

**Discussion by Chienyuuu6** [1101-GS4719-C/Python-Community-GS4719-C#1288] [General]
我去年十月在陽明交通大學參加了一場有關「AI與藝術」的TEDxSalon活動，主要是在探討AI藝術創作與人類藝術創作的arguments 。

老實說到了現在我對這個議題還是蠻拉扯的，究竟AI藝術會不會取代藝術家？

理性的我覺得大部分一件藝術品，外表美感還是第一順位，倘若我們不知道創作者是AI還是人類其實也分不出來，且AI藝術相對創作成本低，只要給AI一些關鍵字跟描述就可以產出各種風格的作品。

感性的我卻覺得人類藝術家在創作的過程，是有很多不為人知的故事、意涵，當觀賞時瞭解其相關的意義後，對於這件作品是有價值昇華的效果！

我得到最後的結論是：AI藝術家和人類藝術家是能夠共存的，畢竟欣賞藝術的人青菜蘿蔔各有各的喜好，像我還是比會被人類藝術家創作的過程所吸引～

不知道各位有什麼看法？

當時的活動相關資訊：https://www.instagram.com/p/CjN3-zmBWQq/?igshid=YmMyMTA2M2Y=

還有這是當時參加活動，主辦方用AI藝術創作的明信片～
![image](https://user-images.githubusercontent.com/126646329/234763088-83631221-e6c2-44c0-85a2-04dc4c85eb8b.jpeg)
![image](https://user-images.githubusercontent.com/126646329/234763145-30e9ec3c-75b1-4f47-b1e8-b6cd9f288641.jpeg)

**Comment by KevinCheng42** [comment]
我很認同作者的觀點，我覺得未來AI藝術家和人類藝術家是會同時存在的，並且兩者不會有取代或被取代的關係，因為大家對於品味欣賞藝術的觀點不同，有些人如我自己喜歡了解藝術家在創作時背後的故事，有些人則著重於作品或創作本身，秉持著類似於「作者已死」的觀點欣賞藝術～

**Comment by YoYooHuang** [comment]
我認為不會。因為AI的訓練都是透過人類所給予的資料進行訓練，而這些資料都是人類事先創造出來的。人類跟AI的差別就在於“創意”，這是人類不可取代的一項技能。所以我認為如果人類能夠繼續創新，才能不被AI取代。

**Comment by lyien1021** [comment]
我覺得藝術很難被AI完全取代，因為藝術是有生命的，像是莫內和梵谷的印象派，那種筆觸真的很難模仿出來，又或是達利的超現實主義，那些富含創意的作品我認為是有無法被取代的價值與意義。

**Comment by a3240281370** [comment]
我覺得藝術是人類創造力和情感表達的產物，涉及到獨特的思考、情感和經驗。儘管AI可以生成新的藝術作品，但它缺乏情感、主觀性和個人獨特性，這些都是人類藝術家所擁有的獨特特質。儘管如此，AI在藝術領域的應用也有其價值，它可以作為工具和合作夥伴，幫助藝術家實現他們的創意想法。AI可以用於藝術品創作的生成、數據分析和視覺效果等方面，提供新的創作方式和技術手段。
